src/horizon_api.ts

```diff
@@ -63,6 +63,16 @@ export namespace Horizon {
     buying_liabilities: string;
     selling_liabilities: string;
   }
+  export interface BalanceLineLiquidityPool {
+    liquidity_pool_id: string;
+    asset_type: AssetType.liquidityPoolShares;
+    balance: string;
+    limit: string;
+    last_modified_ledger: number;
+    is_authorized: boolean;
+    is_authorized_to_maintain_liabilities: boolean;
+    is_clawback_enabled: boolean;
```

Comment:
a pool share trustline can be sponsored so I think we need a `sponsor?: string` here

Reply:
Nice catch, will be fixed by #702 :+1: